### PR TITLE
fix #81520 dilbilgisi.net

### LINKED ISSUE #81520: www.dilbilgisi.net
### Issue URL (Ads)
 [https://www.dilbilgisi.net/fiilimsi-konu-anlatimi/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.dilbilgisi.net%2Ffiilimsi-konu-anlatimi%2F)
### Comment
Username: @gork7777
> Ads

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6.1
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode: | disabled
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish
Userscripts: | https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#81520

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -3332,7 +3332,7 @@ gidahatti.com##a[href^="https://tr.hach.com"]
 ||cdn.xix*.com/bnr^
 ||imagizer.imageshack.us/a/img922/8810/wHDdVD.gif
 evdesifa.com##.post .code-block
-hwp.com.tr,konupara.com,lafmacun.net,finanskocaeli.com##.code-block
+dilbilgisi.net,hwp.com.tr,konupara.com,lafmacun.net,finanskocaeli.com##.code-block
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:3335]: also blocks useful element `.code-block-16`

let's use `dilbilgisi.net##.code-block:not(.code-block-16)`

Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:3335]: also blocks useful element `.code-block-16`

let's use `dilbilgisi.net##.code-block:not(.code-block-16)`

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -107,6 +107,7 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||hankyung.com/resource/js/w/view.gtmet.js
 ||cdn.thunderhead.com/one/rt/js/one-tag.js
 ||discord.com/api/v*/science
+/flying-analytics/js/analytics.js
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/specific.txt:110]: please add $domain

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 2 above)
Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:110]: It looks popular
https://publicwww.com/websites/%22%2Fflying-analytics%2Fjs%2Fanalytics.js%22/

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply TurkishFilter/sections/specific.txt:3335]: what about `div[class^="code-block code-block-"][style*="margin: 8px auto; text-align: center;"]`?

File: SpywareFilter/sections/specific.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply SpywareFilter/sections/specific.txt:110]: @Alex-302 so to general_url.txt ?

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:110]: @wout-codes @Sergey-Lyapin  Yes

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply TurkishFilter/sections/specific.txt:3335]: Negation doesn't worth it in some cases.

**Pull request merged**